ResourceStatusSystem/Client/SiteStatus.py

```diff
@@ -179,7 +179,27 @@ def getUsableSites( self, siteNamesList ):
     """
 
     if not siteNamesList:
-      return S_ERROR(DErrno.ERESUNK, 'siteNamesList is empty')
+      if self.rssFlag:
+        activeSites = self.rsClient.selectStatusElement( 'Site', 'Status', status = 'Active', meta = { 'columns' : [ 'Name' ] } )
+        if not activeSites['OK']:
+          return activeSites
+
+        degradedSites = self.rsClient.selectStatusElement( 'Site', 'Status', status = 'Degraded', meta = { 'columns' : [ 'Name' ] } )
+        if not degradedSites['OK']:
+          return degradedSites
+
+        return activeSites['Value'] + degradedSites['Value']
```

Comment:
Still, something strange: at line 224 you do:
```
return S_OK( siteStatusList )
```
while here you do
```
return activeSites['Value'] + degradedSites['Value']
```
which is a simple `list`. It should be `S_OK` also here I think.